src/ansys/fluent/core/workflow.py

```diff
@@ -504,19 +527,52 @@ def insert_next_task(self, command_name: str):
         Raises
         ------
         ValueError
-            If the command name does not match a task name.
+            If the Python name does not match the next possible task names.
         """
-        if command_name not in self.get_next_possible_tasks():
+        # The next line populates the Python name map for next possible task
+        self._get_next_python_task_names()
+        if task_name not in self.get_next_possible_tasks():
             raise ValueError(
-                f"'{command_name}' cannot be inserted next to '{self.python_name()}'. \n"
+                f"'{task_name}' cannot be inserted next to '{self.python_name()}'. \n"
                 "Please use 'get_next_possible_tasks()' to view list of allowed tasks."
```

Comment:
```suggestion
                "Use 'get_next_possible_tasks()' to view list of allowed tasks."
```
Omit "please" in technical doc.